sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/CheckpointFileManager.scala

```diff
@@ -274,7 +275,8 @@ class FileSystemBasedCheckpointFileManager(path: Path, hadoopConf: Configuration
         throw QueryExecutionErrors.renameSrcPathNotFoundError(srcPath)
       } else {
         val e = QueryExecutionErrors.failedRenameTempFileError(srcPath, dstPath)
-        logWarning(e.getMessage)
+        logWarning(log"Failed to rename temp file ${MDC(TEMP_PATH, srcPath)} to " +
```

Comment:
To output variables from `e.message` to the log
https://github.com/apache/spark/blob/8c2c0fb43a9d3c5bffcf33aeb3354c01fe6b26cd/sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala#L1650-L1657
https://github.com/apache/spark/blob/8c2c0fb43a9d3c5bffcf33aeb3354c01fe6b26cd/common/utils/src/main/resources/error/error-conditions.json#L1304-L1309